tests/acceptance/category.feature

```diff
@@ -0,0 +1,83 @@
+Feature: category
+  In order to manage category in the web
+  As an owner
+  I need to create modify trash publish and Unpublish category
+
+  Background:
+    Given Joomla CMS is installed
+    When Login into Joomla administrator with username "admin" and password "admin"
+    Then I see administrator dashboard
+
+  Scenario: Verify available tabs in Category
+    Given There is an article category link
+    When I check available tabs in category
+    Then I see available tabs "Category", "Publishing", "Permissions" and "Options"
+
+  Scenario: Create new category
+    Given There is an article category link
+    When I fill mandatory fields for creating Category
+      |     Title     |
+      |   Category_1  |
+      |   Category_2  |
```

Comment:
Because Category_1 is trashed then we require a one more category for scenario Create menu item for articles belonging to a specific Category, ACL and Language setting etc..